composer.json

```diff
@@ -14,11 +14,11 @@
         "spryker/glossary": "^3.0.0",
         "spryker/guzzle": "^2.1.0",
         "spryker/kernel": "^3.33.0",
-        "spryker/oms": "^6.0.0 || ^7.0.0 || ^8.0.0 || ^10.0.0 || ^11.0.0",
+        "spryker/oms": "^7.3.0 || ^8.0.0 || ^10.0.0 || ^11.0.0",
         "spryker/propel-orm": "^1.6.0",
         "spryker/quote": "^2.14.0",
-        "spryker/refund": "^4.0.0 || ^5.0.0",
-        "spryker/sales": "^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0 || ^10.0.0 || ^11.0.0",
+        "spryker/refund": "^5.1.0",
+        "spryker/sales": "^8.2.1 || ^10.0.0 || ^11.0.0",
```

Comment:
The reason was Travis builds in suite-nonsplit, which checked the low versions, and fell with errors like: "for this package version is 5.1.0 minimum, but 4.0.0 is indicated". Because of this for some packages minimum versions were increased.